style.css

```diff
@@ -241,3 +254,177 @@ ul {
 .more-icon {
   margin-right: 2rem;
 }
+
+.portfolio-footer {
+  margin-top: -10rem;
+  border-top-left-radius: 7rem;
+  background-color: #6070ff;
+  color: #ffff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 45rem;
+  background-image: url('images/contact-form-background.png');
+  background-size: contain;
+  background-position: right;
+  background-repeat: no-repeat;
+}
+
+.footer-form {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  width: 80%;
+  align-items: center;
+}
+
+.footer-description {
+  text-align: center;
+  color: #ebebff;
+  line-height: 1.4rem;
+}
+
+.form-input {
+  height: 2.5rem;
+  width: 100%;
+  padding-left: 1rem;
+  border-radius: 0.5rem;
+  border: none;
+}
+
+.form-textarea {
+  width: 100%;
+  height: 10rem;
+  padding-left: 1rem;
+  padding-top: 1rem;
+  border-radius: 0.5rem;
+  border: none;
+}
+
+.form-button {
+  background-color: white;
+  color: #6070ff;
+  padding: 0.5rem 1rem;
+  border: none;
+  border-radius: 0.2rem;
+  align-self: flex-start;
+}
+
+@media (min-width: 1024px) {
+  body {
+    font-size: 20px;
+  }
+
+  #page-header {
+    height: 3.5rem;
+    background-color: #ffff;
+    align-items: center;
+    padding: 0 5rem;
+    box-shadow: 0 0 1rem #a7aeff;
+  }
+
+  .header-li a {
+    color: #344563;
+    font-weight: 400;
+  }
+
+  #hamburger-menu {
+    display: none;
+  }
+
+  #header-ul {
+    display: flex;
+  }
+
+  .header-li {
+    margin: 0 1rem;
+  }
+
+  #section-1 {
+    background-image: url('images/header-background-desktop.png');
+    background-size: auto;
+    padding: 5rem 25rem;
+  }
```

Comment:
- [ ] Kindly reduce the padding left and right to make your page looks more like the desktop version of the Figma template. There is a lot of whitespace on the sides of this section.
 - Your version:
 - 
<img width="1113" alt="image" src="https://user-images.githubusercontent.com/61919617/214934218-69913ba2-a702-4596-a57a-92864311b846.png">

 - Figma template:
 - 
<img width="984" alt="image" src="https://user-images.githubusercontent.com/61919617/214934283-2a944fd0-5c3a-408f-b778-e1fbc6c1d615.png">